src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -190,9 +190,10 @@ void CreateRequestQueue::RemoveDisconnectedClientRequests(
 
   for (auto it = fulfilled_requests_.begin(); it != fulfilled_requests_.end();) {
     if (it->second && it->second->client == client) {
-      fulfilled_requests_.erase(it);
+      fulfilled_requests_.erase(it++);
```

Comment:
Actually, I'd push for using the returned iterator here, since that's considered the idiomatic pattern (even included in the `map.erase()` docs as an example):
```suggestion
      it = fulfilled_requests_.erase(it);
```

Reply:
yup `it = fulfilled_requests_.erase(it);` definitely much better code. 
However, it's unfortunate that this absl map.[erase returns void](https://github.com/abseil/abseil-cpp/blob/master/absl/container/flat_hash_map.h#L223) :'(

Reply:
Ah I forgot that this is an `absl::flat_hash_map`!